docs/source/concept_models/vivarium_ciff_sam/SAM_compartmental_diagram

```diff
@@ -0,0 +1 @@
3.2.2 Chrome/83.0.4103.100 Electron/9.0.3 Safari/537.36" etag="Rh9ZbrrhHHdn4Y7jEz_a" version="13.2.2" type="device"><diagram id="suxCMKjWDzoVuUeKPbTA" name="Page-1">7Vxbc5s4FP41nuk+pIMkbn5MkzQ7281ud5x2u/uSIbZss8XIBZzY/fUrgQQICZs4BpskfUjhSOhyjr5PR0eSB+hisb6OvOX8hkxwMIDGZD1AlwMIAUAO/Y9JNpnEss1MMIv8SSYyCsHI/4n5l0K68ic45rJMlBASJP5SFo5JGOJxIsm8KCKPcrYpCSaSYOnNsCIYjb1Alf7tT5J5JnUto5D/iv3ZXNQMDJ6y8ERmLojn3oQ8lkToaoAuIkKS7GmxvsABU56sl481qXnDIhwmTT6Yfo/u7L+8r7Mf19/uP316+Bf692duVsqDF6x4h3ljk43QAJ5QhfBXEiVzMiOhF1wV0g8RWYUTzKox6FuR53dCllQIqPA/nCQbbl1vlRAqmieLgKeqXeG9i8kqGuMt7RdDwotmONmSj3eL9aVUAVfUNSYLnEQbmiHCgZf4D7LxPT6GZnm+Qs30gWtar/WvJHR+XppfPl/fmr/dfV//HP9pnKGXofXDaZN/+pn4tCnQ4Owx5Mjh1AGcoVxCZnX+UcUkeSsaWWlbL0tW+oOEZ0zpUUg15Mdzqn1oB1R5H+4j+jRjT2OP9cBUDFqYi+n+ce4neLT0UjU/UupsaJoHHCV4vVXpPNW0JOXZXJePBYkJ9c5L/CWyPWfIa5VpOj0f87Ah0wi9d0812uYIg/Zd7wfmmvMo8jalDEtGInE9FUEkwwmAymT7tPz0IWtBw6/dCixbZj6ojJlHL078cFZHd+jodAeGO/nO7ZLvxFxV0uGC4sYL/GRzdGUJMs6HF1KUZRnOe2ipCoNtKQydBlFNvHRafy5roVZYSyUKB0qWNA23U6JQHdkvYRJhL2GekXFzfqMSxruSNF7dC/FqLYS0ISX5L2oJQjIlqSrGJCBRWrn9Y0WyDAh79r1tl0XZt+DsolRPVoJcapXa1FEoj7EOqA3ZkpGR7TbituqkcTCoqtRmLKijsVA0RTuYyOqIk4h8xxfcZCEJGWinfhBURJQnZyF9HVO1YSr/wNTl02X5OU9Y+JNJinid/mULHYIw5ZUIUicXR2OA9rjSfklcKUI97ZOlKZOl2y1ZAnWGu92LK5PDcuUUDMfDocqV/WdK0zwyUwJ1gvQ1Br2jCkR6m75cTgW7IxRml6QKdLFQxVS5aOmFkmEEehhSzuKUMs9pBgCXaxVaV2ufISai4E97TKl3xQSp77TAKdbywZDWlNfuLZg1wvt4WWpNZcyw9hiNoD9FSAd9armPlzvgr1HKPlVRipN8P7kqY1sPIx2UquVJzaTFsb81GFzFOK77sqr32mJG5zd304gsMk6vb8pJwjrfozgQxqshXOjYKsi7RbnqvHaI8r0gbjQS1Q34FCkSnksZ6AhlzsQ2lOl006DqOmzW+S5PQFgrQK1v1ysBqjwZI0MNCZmwS6Cauk2yzEIT/0ELSmaaM65lhkquaAWXMrq1U9jUG8tlz3HwgJm1njeZ6yZSlQCEOO2oLD1U3/foZ0El9L8aNqmf73apQOqrHnPtLyDgEMgwEGHcEgyAqwmLotZg0PcdeuHl7944sw8dBnie4i1F8dlqXDv5FWvhfKfEUNN2r5O7if8jUw4bH3uzxFTjWSNdPGS3bsHRdWs6xmnpVnV460IQcDt31zrDfL3Hpp1odu+9M1ID0eYa2qdfUreU+WWpFz31Fn6wyT5fkJDEqXWkLIWjbej87AG0qifQLKohJk2PV+VvQmNWqjMquWTPrGEW05HFZrYdeUGeVwyTvYqBRTGZYfIUn/1lCqeSzEBW2k32yuNERXpaWSVTUXJmqbzkdBZlOdIBz8Qgq6TkvzJp6q6yhMKHlcSZl8FEwsmwqr4sS+TeLEssoY6lpLhjcoFNJjQqjeeIa6L4PCnXeQFLiwGzyMlYXRSzKeR2YTsO1DzJLCVxwBZWLKVR4ObyWan66oBIX/NRURbKY5XnUwZ1jc94kquUQ3hk7k467TR+IA5+NqBT8Eanb3T6RqdvdHpCdJrHcE6FTtVDturyNpycs5P8TIuBF8f+uKHTvnMlKZ13UnstZM/cd66uCMRZwLyIbA2t7DurBUGwvaCWN7At+Hxj4bWffGPj+b0BbP7+T/ruQvF+ueYDPn3ZlF4+48invWBA2o6IiukbxymONUYQOtAYgY7+4GlnY6TJ3YaeAxoM4fsdWm5qLoCcXUW1bTBTMVj0VIf2JGfLQ2+RmEYFolCdO3WRnfbmTjVCWWe52sjOq7Bc7uWcjOXUeGed5fp1LOjQlgPwxCxnn/jVvbbPUCJHmfqquq6Z+p564wY5SFtPsxs0mnZ2fIfGPo0zukcbKRboaqRY4DkjRdPOjkeKWB6WQ4p36V7jq+B4W17UmkA9s9tpSMJRHeJjALfZ4fq2UWxWjGPadJ1uFP+sRojWLHuqB7W3l9s2AtXLyZ8j/ECV7ZMwVXwyf/TUe3svE5AWkPlQjP8SIDtGpHoG5rY4W/XKrQNM1SXu1jyuug5VCbPnIR+rsg0JKleHmhKf7dpby2mZ6NwmTmnPTaWoGMH9bGW5cHtBbRtL95MZPTkBO3idx1+3nG7vx/FX+YcBTPvYx19d3XWNU3XGd+4xidswrd90tWUKtFGliJaZa6iuaMu/C6A92/lu9Pa7AE9DK6jsGVri8PrRfhegyWZ+f8AKuwKrK5nRAR2DFSpWu90Lqm/X0hsC1XaODdS3u1V9cy5H/b9b5Vbv+6txpW6dy6Ea6fXvRq8m8G5V7GGqrNRpHGnYJI50Ku5D6xutFePYQylAvmdIQ5mJthfbtuehi0b1ZBp6pXNQ7wMc8n6WLZzfFuagAT+CXoJMcfgcXf0P</diagram></mxfile>
```

Comment:
I don't think this file needs to go in the repo. Do let me know if I'm wrong about that, though.